content/articles/2017-12-14-styler-1-0-0.Rmd

```diff
@@ -0,0 +1,237 @@
+---
+title: styler 1.0.0
+author: Lorenz Walthert, Kirill Müller
+date: '2017-12-14'
+slug: styler-1.0.0
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/mpdIPhYqZ4Y
+  author: Heng Films
+description: >
+  styler, a package that can format your code according to the tidyverse style guide, is now on CRAN.
+
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = TRUE, comment = NA, promt = TRUE)
+knitr::knit_engines$set(list(
+  styler = function(options) {
+    options$comment <- ""
+    knitr::engine_output(
+      options,
+      
+      c("# Before", options$code),
+      c("# After", styler::style_text(options$code))
+    )
+  }
+))
+
+```
+
+
+We're pleased to announce the release of [styler 1.0.0](https://github.com/r-lib/styler/releases/tag/v1.0.0). styler is a source code formatter - a package to format R code according to a style guide. We've implemented the [tidyverse style guide](http://style.tidyverse.org), so you can format your code according to it. A coherent style is important for consistency and legibility. Just as it is important to putspcesbetweenwords. 
+You can install styler from CRAN:
+
+```{r, eval = FALSE}
+install.packages("styler")
+```
+
+styler can style text, single files, packages and entire
+R source trees with the following functions:
+
+* `style_text()` styles a character vector.
+* `style_file()` styles R and Rmd files.
+* `style_dir()` styles all R and/or Rmd files in a directory.
+* `style_pkg()` styles the source files of an R package.
+* An RStudio Addin that styles the active R or Rmd file, the current 
+  package or the highlighted code.
+
+Note that all functions except `style_text()` modify *in place*, that is, they overwrite the source. `style_file()` for example modifies the files it is given without creating a backup copy. We therefore strongly advice to use version control in conjunction with styler and consider the implications this might have on your workflow.
+
+## styling options
+
+A distinguishing feature of styler is its flexibility. The following options are available:
+
+* `scope`. What is to be styled, e.g. just spacing or also indention, line breaks, tokens?
+* `strict`. How strict should be rules be applied?
```

Comment:
Technically, the should be "strictLY", i.e. an adverb.

The first "be" should presumably be a "the".

Reply:
Agree.